tss-esapi-sys/README.md

```diff
@@ -41,9 +41,70 @@ available, feel free to raise a Pull Request to add it or to use build-time
 generation of bindings. All the committed bindings **MUST** be generated from
 the library version found under the `vendor` submodule.
 
+## Bundling TPM-TSS
+
+tpm-tss is used by this library to communicate with TPMs. If this library
+is not available on your system you may optionally bundle (vendor) tpm-tss
+during builds. tpm-tss can be provided from a local source path with the
+environment variable `TPM_TSS_SOURCE_PATH` or it will be retrieved from
+github during the build. The version to retrieve can be controlled by setting
+the `TPM2_TSS_SOURCE_VERSION` environment variable.
+N.B. On windows it might be necessary to manually create the VERSION file
+when a local source is being used.
+
+To enable this feature:
+
+```bash
+cargo build --features=bundled
+```
+
+```bash
+TPM2_TSS_VERSION="4.1.3" cargo build --features=bundled
+```
+
+```bash
+TPM_TSS_SOURCE_PATH=/path/to/tpm-tss cargo build --features=bundled
+```
+
+If using this feature from an external project
+
+```
+tss-esapi-sys = { version = "...", features = "bundled" }
```

Comment:
```suggestion
tss-esapi-sys = { version = "...", features = ["bundled"] }
```

Not sure if a bare string is accepted but I think it need to be an array :thinking: